app/services/pages/loader.rb

```diff
@@ -0,0 +1,12 @@
+module Pages
+  class Loader
+    def self.load
+      Pages::Manager.add_page({ :controller => :smart_proxies, :action => :show }, "smart_proxies/show") do |page|
```

Comment:
`Pages::Manager` keeps track of pages that may be extended, here the page for smart_proxy/show is registered for extension